frontend/src/features/resources/ManageResourcesPage.jsx

```diff
@@ -0,0 +1,559 @@
+import React, { useMemo, useRef, useState, useEffect } from "react";
+import { getAuth } from "firebase/auth";
+import axios from "axios";
+import Papa from "papaparse";
+import toast from "react-hot-toast";
+import { useParams } from "react-router-dom";
+
+// Your project components
+import ScreenContainer from "../../components/ScreenContainer/ScreenContainer";
+import TopBar from "../../components/TopBar/TopBar";
+
+/**
+ * ManageResourcesPage
+ * - Keeps your existing header (TopBar)
+ * - LEFT: nested DaisyUI dropdowns (collections → sub-items) with a + button to upload files into each
+ * - RIGHT: searchable table + preview of files you uploaded via the left pane
+ * - Also preserves your CSV bulk import flow into `resources` (shown below header for reference)
+ */
+export default function ManageResourcesPage() {
+  const { scenarioId } = useParams();
+
+  // ==== existing CSV import state ==== //
+  const csvInputRef = useRef(null);
+  const [resources, setResources] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const auth = getAuth();
+        const user = auth.currentUser;
+        if (!user) {
+          toast.error("You must be logged in to view resources.");
+          return;
+        }
+        const idToken = await user.getIdToken();
+        const response = await fetch(
+          `http://localhost:3000/api/resources/scenario/${scenarioId}`,
+          {
+            headers: { Authorization: `Bearer ${idToken}` },
+          }
+        );
```

Comment:
for consistencies sake, it'd be better to use `axios.get` here rather than `fetch`. Also make sure the url is only the part from the `/api` onwards.